src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -305,23 +316,26 @@ class Lambda extends RequestStreamHandler {
       val folderRow = folderInfo.folderRow
       val entity = folderInfo.entity.get
 
-      val updateEntityRequestWithNoUpdates =
-        UpdateEntityRequest(entity.ref, None, None, structuralObject, folderInfo.securityTag.get, None)
-
-      val potentiallyUpdatedTitleRequest =
-        if (folderRow.title.getOrElse("") != entity.title.getOrElse(""))
-          updateEntityRequestWithNoUpdates.copy(titleToChange = folderRow.title)
-        else updateEntityRequestWithNoUpdates
+      val potentialNewTitle = folderRow.title.getOrElse("")
+      val potentialNewDescription = folderRow.description
 
-      val potentiallyUpdatedTitleOrDescriptionRequest =
-        if (folderRow.description.getOrElse("") != entity.description.getOrElse(""))
-          potentiallyUpdatedTitleRequest.copy(descriptionToChange = folderRow.description)
-        else potentiallyUpdatedTitleRequest
+      val titleHasChanged = potentialNewTitle != entity.title.getOrElse("")
+      val descriptionHasChanged = potentialNewDescription.getOrElse("") != entity.description.getOrElse("")
```

Comment:
Can you just compare the descripton directly `potentialNewDescription == entity.description`? I don't think you need the getOrElse.
You could also do `potentialNewTitle == entity.title` and then you only need one `getOrElse` on line 321. What do you think?